ethereum/p2p/src/main/java/tech/pegasys/pantheon/ethereum/p2p/discovery/internal/PeerDiscoveryController.java

```diff
@@ -135,6 +136,9 @@
 
   private RecursivePeerRefreshState recursivePeerRefreshState;
 
+  private final NodePermissioningController nodePermissioningController =
```

Comment:
In this implementation, we aren't using the NodePermissioningController for any permissioning.